apps/deploy-web/src/components/new-deployment/CreateLease.tsx

```diff
@@ -159,48 +205,10 @@ export const CreateLease: React.FunctionComponent<Props> = ({ dseq }) => {
         category: "deployments",
         label: "Create lease"
       });
-    } catch (error) {
-      // Rejected transaction
+      setIsLeaseCreated(true);
```

Comment:
Out of curiousity, why don't we just call `sendManifest` here instead of using the `useWhen`?

Reply:
You may be right) I'll check this once again

Reply:
so yea, you're right, changed that